common/lib/xmodule/xmodule/modulestore/tests/django_utils.py

```diff
@@ -213,10 +245,23 @@ def my_test(self):
                 ...
 
     """
-
     MODULESTORE = functools.partial(mixed_store_config, mkdtemp_clean(), {})
     CONTENTSTORE = functools.partial(contentstore_config)
     ENABLED_CACHES = ['default', 'mongo_metadata_inheritance', 'loc_cache']
+
+    # List of modulestore signals enabled for this test. Defaults to an empty
+    # list. The list of signals available is found on the SignalHandler class,
+    # in /common/lib/xmodule/xmodule/modulestore/django.py
+    #
+    # You must use the signal itself, and not its name. So for example:
+    #
+    # from xmodule.modulestore.django import SignalHandler
+    #
+    # class MyPublishTestCase(ModuleStoreTestCase):
+    #     ENABLED_SIGNALS = [SignalHandler.course_published, SignalHandler.pre_publish]
+    #
+    ENABLED_SIGNALS = []
```

Comment:
And, if the concern is misspelling, you could just make the lookup fail hard if the name doesn't exist.